content/support-training.html

```diff
@@ -209,6 +209,13 @@ <h2>Commercial support</h2>
               <img src="../assets/commercial-support-logos/xamira_networks.png" alt="xamira networks">
             </a>
           </div>
+
+        <div class="col-md-3 col-sm-6 doc-content tc-col">
+            <a href="https://www.asyncdatado.com/" class="btn btn-training btn-default btn-large" target="_blank">
```

Comment:
We try to maintain alphabetical ordering on this page (although I see that "acend" is already incorrectly sorted as well). Could you please update the ordering to: acend -> asyncdatado -> container solutions in the beginning?